extension/js/common/composer.ts

```diff
@@ -1653,8 +1714,8 @@ export class Composer {
     }, this.getErrHandlers(`focus on recipient field`))).children().click(() => false);
     this.resizeInputTo();
     this.attach.initAttDialog('fineuploader', 'fineuploader_button');
-    this.attach.setAttAddedCb(async () => this.setInputTextHeightManuallyIfNeeded());
-    this.attach.setAttRemovedCb(() => this.setInputTextHeightManuallyIfNeeded());
+    this.attach.setAttAddedCb(async () => { this.setInputTextHeightManuallyIfNeeded(); this.resizeComposeBox(); });
```

Comment:
when there is more than one command, please format it as multiline (max one statement; per line).

Reply:
Okay, got it